packages/core/content-type-builder/server/utils/attributes.js

```diff
@@ -67,6 +67,19 @@ const formatAttribute = (key, attribute) => {
     };
   }
 
+  if (attribute.type === 'customField') {
+    const customField = strapi.container.get('custom-fields').get(attribute.customField);
+
+    if (!customField) {
+      throw new Error(`Could not find Custom Field: ${attribute.customField}`);
+    }
```

Comment:
We could add a test case for this error

Reply:
On second thought I think it might make more sense to move this error (and test case) to the registry itself so we don't have to do checks like this every time we use the API. What do you think?